app/presenters/monitoring-stations/view-monitoring-stations.presenter.js

```diff
@@ -0,0 +1,135 @@
+'use strict'
+
+/**
+ * Formats the monitoring station details and related licences data for the
+ * `/monitoring-stations/{monitoringStationId}` page
+ * @module ViewMonitoringStationsPresenter
+ */
+
+const { formatAbstractionPeriod, formatLongDate } = require('../base.presenter.js')
+
+/**
+ * Formats the monitoring station details and related licences data for the
+ * `/monitoring-stations/{monitoringStationId}` page
+ *
+ * @param {string} auth - The auth object taken from `request.auth` containing user details
+ * @param {GaugingStationModel[]} monitoringStation - The monitoring station and associated licences data returned
+ * by FetchMonitoringStations
+ *
+ * @returns {object} monitoring station and licence data needed by the view template
+ */
+function go (auth, monitoringStation) {
+  const formattedLicences = _formatLicences(monitoringStation.licenceGaugingStations)
+  const sortedLicences = _sortLicences(formattedLicences)
+  const groupedLicences = _groupLicences(sortedLicences)
+
+  return {
+    gridReference: monitoringStation.gridReference,
+    hasPermissionToManageLinks: _checkPermissions(auth, 'manage_gauging_station_licence_links'),
+    hasPermissionToSendAlerts: _checkPermissions(auth, 'hof_notifications'),
+    licences: groupedLicences,
+    monitoringStationId: monitoringStation.id,
+    monitoringStationName: monitoringStation.label,
+    pageTitle: _createPageTitle(monitoringStation.riverName, monitoringStation.label),
+    stationReference: monitoringStation.stationReference,
+    wiskiId: monitoringStation.wiskiId
+  }
+}
+
+function _formatLicences (licenceDetails) {
+  return licenceDetails.map((licenceDetail) => {
+    return {
+      abstractionPeriod: _formatLicenceDetailsAbstractionPeriod(licenceDetail),
+      alertType: _alertType(licenceDetail),
+      alertUpdatedAt: _alertedUpdatedAt(licenceDetail),
+      createdAt: licenceDetail.createdAt,
+      lastUpdatedAt: licenceDetail.statusUpdatedAt,
+      id: licenceDetail.licence.id,
+      licenceRef: licenceDetail.licence.licenceRef,
+      restrictionType: licenceDetail.restrictionType === 'flow' ? 'Flow' : 'Level',
+      threshold: `${licenceDetail.thresholdValue} ${licenceDetail.thresholdUnit}`
+    }
+  })
+}
+
+function _alertType (licence) {
```

Comment:
With our private functions, our coding convention is to put them alphabetically. 

You wouldn't have known this though as we need to update our coding conventions to include this  <img src="https://media2.giphy.com/media/xT5LMpBWU96Yivdfeo/giphy.gif"/>